src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import useStyles from './styles';
+import { RadioBaseProps } from './base';
+import Radio from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+interface RadioButtonProps extends RadioBaseProps {
+  children: string;
+  disabled?: boolean;
+  checked?: boolean;
+}
+
+const RadioButton: React.FC<RadioButtonProps> = ({
+  children,
+  disabled,
+  checked,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <FormControlLabel
+      value="end"
+      label={children}
+      className={classes.RadioLabel}
+      control={
+        <Radio
+          color="primary"
+          disabled={disabled}
+          checked={checked}
```

Comment:
I believe checked should automatically be handled by `Radio` we won't need to pass checked as a prop.

Reply:
@S-ayanide so,  I need to remove checked?

Reply:
Yes, it should auto infer